python/cucim/src/cucim/skimage/measure/tests/test_regionprops.py

```diff
@@ -601,6 +601,15 @@ def test_props_to_dict():
                    'bbox+0': cp.array([0]), 'bbox+1': cp.array([0]),
                    'bbox+2': cp.array([10]), 'bbox+3': cp.array([18])}
 
+    regions = regionprops(SAMPLE_MULTIPLE)
+    out = _props_to_dict(regions, properties=('coords',))
+    coords = np.empty(2, object)
+    coords[0] = cp.array((cp.arange(10), cp.arange(10))).T
```

Comment:
equivalent, but seems a bit clearer to me

```suggestion
    coords[0] = cp.stack((cp.arange(10),) * 2, axis=-1)
```

another option would be
```Python
coords[0] = cp.stack(cp.where(SAMPLE_MULTIPLE == 1), axis=-1)
```